src/cpp/src/icontinuous_batching.cpp

```diff
@@ -146,4 +201,14 @@ void ContinuousBatchingPipeline::IContinuousBatchingPipeline::stream_tokens(
     const auto tokens = generation_outputs.begin()->second.generated_ids;
     streamer_ptr->write(tokens);
 }
+
+GenerationHandle ContinuousBatchingPipeline::IContinuousBatchingPipeline::add_request(uint64_t request_id,
```

Comment:
probably, we need to move implementation of this method to `ContinuousBatchingImpl` ? as for other `add_request` methods

Reply:
Moved to `ContinuousBatchingImpl`, but in this case this method also needs to be implemented in `SpeculativeDecodingImpl` and `PromptLookupImpl`.

Reply:
Okay, moved this back to `IContinuousBatchingPipeline`.